Docs: `poetry` example on `build.jobs` section (#9445)
readthedocs/readthedocs.org: docs/user/build-customization.rst

         - npm install -g jsdoc

Install dependencies with Poetry
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

> In case the project is managing Python dependencies with `Poetry <https://python-poetry.org/>`__,
> it can use ``post_create_environment`` user-defined job to install them while building the documentation.

Reviewer: Just making language a bit clearer.

```suggestion
Projects managed with `Poetry <https://python-poetry.org/>`__,
can use the ``post_create_environment`` user-defined job to use Poetry for installing Python dependencies.
```
